examples/cmd/benchmark.go

```diff
@@ -123,15 +127,26 @@ func runBenchmark(cmd *cobra.Command, _ []string) error {
 		// 	}
 		// }
 	} else {
-		tdf, err := client.CreateTDF(
-			out, in,
-			sdk.WithDataAttributes(dataAttributes...),
-			sdk.WithKasInformation(
+		opts := []sdk.TDFOption{sdk.WithDataAttributes(dataAttributes...), sdk.WithAutoconfigure(false)}
+		if insecurePlaintextConn || strings.Contains(platformEndpoint, "http://") {
+			opts = append(opts, sdk.WithKasInformation(
 				sdk.KASInfo{
-					URL:       "http://" + "localhost:8080",
+					URL:       "http://localhost:8080",
```

Comment:
See note above about platformEndpoint maybe being back in play. Maybe file a follow-up issue about getting examples working with non-localhost servers

Reply:
can do, ill add it to tech debt